src/sv-pipeline/scripts/downstream_analysis_and_filtering/make_batch_effect_reclassification_table.PCRMinus_only.R

```diff
@@ -34,12 +33,14 @@ import.freqs <- function(freq.table.in){
   out <- data.frame("VID"=dat$VID,"PCRPLUS_AF"=plus.AFs,"PCRMINUS_AF"=minus.AFs)  
   return(out)
 }
+
 #Process failure lists for a single comparison type
-import.fails <- function(all.in,minus.in,prefix){
-  a <- read.table(all.in,header=F,sep="\t")
+import.fails <- function(minus.in,prefix){
+  a <- tryCatch(read.table(minus.in,header=F,sep="\t"), error=function(e){return(as.data.frame(matrix(ncol=2)))})
```

Comment:
Is this to guard against a non-existent file? Would there be any way to check for that explicitly? It seems dangerous to swallow all errors here -- what if there's a transient file system error or something?

Reply:
Yep, this was included to protect against the situation where zero records are marked as having batch effects. This is unlikely to happen in most real-world multi-batch callsets, but did happen when I was testing on the ref panel when split into two random batches.

Good point about the risk of error swallowing here, though. I've removed the `tryCatch()` statement and instead implemented a check of the input file size from within the R script itself:
```
  # Check file size and return an empty dataframe if no records are marked as batch effects
  if(file.info(minus.in)$size == 0){
    a <- as.data.frame(matrix(ncol=2))
  }else{
    a <- read.table(minus.in,header=F,sep="\t")
  }
```

I thought about moving this logic out to the WDL level, and bypass the relevant steps if no variants are marked as batch effects... but decided against it due to the possible situation where one of the two batch effect lists is empty (but the other isn't). Probably cleanest just to check each file within the R script.